src/App.tsx

```diff
@@ -1,9 +1,19 @@
+import { Route, Routes } from "react-router-dom";
 import { Container } from "./App.style";
 
 function App() {
   return (
     <Container>
       <h1>Fruityvice</h1>
+
+      <Routes>
+        <Route path="/" element={<div>RandomFruit component</div>} />
+        <Route path="/allfruits" element={<div>FruitList component</div>} />
+        <Route
+          path="/allfruits/:idFruit"
```

Comment:
does this make sense as a route?
Is not following rest route naming, please read https://restfulapi.net/resource-naming/

I suggest to change `/allfruits` -> `fruits` and `/allfruits/:idFruit` -> `/fruits/:id`

Reply:
thank you it will be amended